docs/components/concepts/clusters.md

```diff
@@ -6,72 +6,88 @@ description: "Learn more about the clusters available in your Camunda 8 plan."
 
-### Auto-pause
+You can choose from three different cluster types:
 
-Free Trial `dev` (or untagged) clusters are automatically paused eight hours after a cluster is created or resumed from a paused state. Auto-pause occurs regardless of cluster usage.
+| Type                                                                                                                                                                                                           | Basic                                                | Standard                      | Advanced                                            |
+| :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :--------------------------------------------------- | :---------------------------- | :-------------------------------------------------- |
+| Usage                                                                                                                                                                                                          | Experimentation, early development, basic use cases. | Production-ready use cases.   | Production with minimal disruption and high uptime. |
+| Uptime Percentage<br/> (Core Automation Cluster<strong>\*</strong>)                                                                                                                                            | 99%                                                  | 99.5%                         | 99.9                                                |
+| Uptime Percentage<br/>(Management Components<strong>\*\*</strong>) &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;                    | 99%                                                  | 99%                           | 99%                                                 |
+| RTO/RPO<strong>\*\*\*</strong><br/>(Core Automation Cluster<strong>\*</strong>) &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | RTO: 8 hours<br/>RPO: 24 hours                       | RTO: 2 hours<br/>RPO: 4 hours | RTO: < 1 hour<br/>RPO: < 1 hour                     |
 
-You can resume a paused cluster at any time, which typically takes five to ten minutes to complete. See [resume your cluster](/components/console/manage-clusters/manage-cluster.md#resume-a-cluster).
+<p><strong>* Core Automation Cluster</strong> means the components critical for automating processes and decisions, such as Zeebe, Operate, Tasklist, Optimize and Connectors.</p>
+<p><strong>** Management Components</strong> means the components responsible for designing and making processes and decisions executable, such as Web Modeler and Console.</p>
```

Comment:
I guess we could remove the Management Components row out of the table, and reference it in the paragraphs below instead? Agreed, this might be clearer?

Reply:
Done - removed 👍